zerver/tests/test_subs.py

```diff
@@ -501,7 +510,6 @@ def test_rename_stream(self) -> None:
             result = self.client_patch('/json/streams/%d' % (stream_id,),
                                        {'new_name': ujson.dumps('stream_name2')})
         self.assert_json_success(result)
-
```

Comment:
It would have been better to not remove this whitespace, since it helped for readability.